<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnAltaMedicamento</class>
 <widget class="QWidget" name="vtnAltaMedicamento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alta Medicamento</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbMedicamento">
     <property name="title">
      <string>Medicamento</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineNomb">
          <property name="statusTip">
           <string>Ingrese nombre del medicamento (solo letrasy númros)</string>
          </property>
          <property name="accessibleDescription">
           <string>textoNumeros</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbMonodroga">
     <property name="title">
      <string>Monodroga</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QTableWidget" name="tablaMonodroga">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>140</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineCantidad">
          <property name="statusTip">
           <string>Ingrese cantidad de la monodroga (solo números)</string>
          </property>
          <property name="accessibleDescription">
           <string>cantidad</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Cantidad</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" alignment="Qt::AlignRight">
         <widget class="QPushButton" name="btnActMon">
          <property name="text">
           <string>Actualizar Monodrogas</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineNomb</tabstop>
  <tabstop>tablaMonodroga</tabstop>
  <tabstop>btnActMon</tabstop>
  <tabstop>lineCantidad</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
